### PR TITLE
rails test now works

### PR DESCRIPTION
**Pull request by audy**
* misc fixture fixes
* added a route for welcome/index
* updated database.yml to be 100% PostgreSQL

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-# Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test
+  database: jurgita_art_test
 
 production:
   <<: *default
-  database: db/production
+  database: jurgita_art_production
```
Comment (agnaite) [inline config/database.yml]: why did you rename the dbs?

Comment (audy) [reply config/database.yml]: Because I changed them to postgres DB names from sqlite db paths.
